qa/suites/orch/cephadm/workunits/task/test_monitoring_stack_basic.yaml

```diff
@@ -61,6 +61,6 @@ tasks:
         curl -s http://${PROM_IP}:9095/api/v1/alerts
         curl -s http://${PROM_IP}:9095/api/v1/alerts | jq -e '.data | .alerts | .[] | select(.labels | .alertname == "CephMonDown") | .state == "firing"'
         # check alertmanager endpoints are responsive and mon down alert is active
-        curl -s http://${ALERTM_IP}:9093/api/v1/status
-        curl -s http://${ALERTM_IP}:9093/api/v1/alerts
-        curl -s http://${ALERTM_IP}:9093/api/v1/alerts | jq -e '.data | .[] | select(.labels | .alertname == "CephMonDown") | .status | .state == "active"'
+        curl -s http://${ALERTM_IP}:9093/api/v2/status
+        curl -s http://${ALERTM_IP}:9093/api/v2/alerts
+        curl -s http://${ALERTM_IP}:9093/api/v2/alerts | jq -e '.data | .[] | select(.labels | .alertname == "CephMonDown") | .status | .state == "active"'
```

Comment:
using the v2 endpoint has definitely improved things but it looks like we still need to update the check itself that it does with jq on the result.

```
2024-08-27T13:51:40.543 INFO:teuthology.orchestra.run.smithi130.stderr:+ curl -s http://172.21.15.143:9093/api/v2/alerts
2024-08-27T13:51:40.548 INFO:teuthology.orchestra.run.smithi130.stdout:[{"annotations":{"description":"You have 1 monitor down. Quorum is still intact, but the loss of an additional monitor will make your cluster inoperable.  The following monitors are down:   - mon.c on smithi143\n","documentation":"https://docs.ceph.com/en/latest/rados/operations/health-checks#mon-down","summary":"One or more monitors down"},"endsAt":"2024-08-27T13:54:52.639Z","fingerprint":"8c3d4cdbac3a93a2","receivers":[{"name":"ceph-dashboard"}],"startsAt":"2024-08-27T13:50:52.639Z","status":{"inhibitedBy":[],"silencedBy":[],"state":"active"},"updatedAt":"2024-08-27T13:50:52.649Z","generatorURL":"http://smithi139.front.sepia.ceph.com:9095/graph?g0.expr=count%28ceph_mon_quorum_status+%3D%3D+0%29+%3C%3D+%28count%28ceph_mon_metadata%29+-+floor%28count%28ceph_mon_metadata%29+%2F+2%29+%2B+1%29\u0026g0.tab=1","labels":{"alertname":"CephMonDown","cluster":"6f8f9c36-6479-11ef-bcd2-c7b262605968","severity":"warning","type":"ceph_default"}},{"annotations":{"description":"Quorum requires a majority of monitors (x 2) to be active. Without quorum the cluster will become inoperable, affecting all services and connected clients. The following monitors are down: - mon.c on smithi143","documentation":"https://docs.ceph.com/en/latest/rados/operations/health-checks#mon-down","summary":"Monitor quorum is at risk"},"endsAt":"2024-08-27T13:54:52.639Z","fingerprint":"cf211c92b380f229","receivers":[{"name":"ceph-dashboard"}],"startsAt":"2024-08-27T13:50:52.639Z","status":{"inhibitedBy":[],"silencedBy":[],"state":"active"},"updatedAt":"2024-08-27T13:50:52.648Z","generatorURL":"http://smithi139.front.sepia.ceph.com:9095/graph?g0.expr=%28%28ceph_health_detail%7Bname%3D%22MON_DOWN%22%7D+%3D%3D+1%29+%2A+on+%28%29+%28count%28ceph_mon_quorum_status+%3D%3D+1%29+%3D%3D+bool+%28floor%28count%28ceph_mon_metadata%29+%2F+2%29+%2B+1%29%29%29+%3D%3D+1\u0026g0.tab=1","labels":{"alertname":"CephMonDownQuorumAtRisk","cluster":"6f8f9c36-6479-11ef-bcd2-c7b262605968","oid":"1.3.6.1.4.1.50495.1.2.1.3.1","severity":"critical","type":"ceph_default"}}]
2024-08-27T13:51:40.549 INFO:teuthology.orchestra.run.smithi130.stderr:+ curl -s http://172.21.15.143:9093/api/v2/alerts
2024-08-27T13:51:40.549 INFO:teuthology.orchestra.run.smithi130.stderr:+ jq -e '.data | .[] | select(.labels | .alertname == "CephMonDown") | .status | .state == "active"'
2024-08-27T13:51:40.553 INFO:teuthology.orchestra.run.smithi130.stderr:jq: error (at <stdin>:1): Cannot index array with string "data"
2024-08-27T13:51:41.270 DEBUG:teuthology.orchestra.run:got remote process result: 5
```
I can see in what it spit out that it's aware of the mon being down, so it seems like the warning has appeared, we just need a good way to check for it. The test run if it / its logs are useful for you https://pulpito.ceph.com/adking-2024-08-27_12:51:47-orch:cephadm-wip-adk-testing-2024-08-26-1718-distro-default-smithi/7875694/

Reply:
thanks for testing it. I modified the query little bit by removing the `.data` since that is not an attribute in the new response.

from my local testing
```
curl -s http://192.168.100.100:9093/api/v2/alerts  | jq -e '.[] | select(.labels | .alertname == "CephHealthWarning") | .status | .state == "active"'
true
```